webapp/src/components/ConfusionMatrix.tsx

```diff
@@ -83,28 +94,45 @@ const ConfusionMatrix: React.FC<Props> = ({
   classOptions,
   predictionFilters,
   labelFilters,
+  confusionMatrix,
 }) => {
   const classes = useStyles();
-
+  const history = useHistory();
+  const location = useLocation();
   const hoverStyle = (theme: Theme) => ({
     backgroundColor: alpha(
       theme.palette.secondary.main,
       theme.palette.action.selectedOpacity
     ),
   });
 
-  const { data: { confusionMatrix } = { confusionMatrix: [] } } =
-    getConfusionMatrixEndpoint.useQuery({
-      jobId,
-      datasetSplitName,
-      ...filters,
-      ...pipeline,
-      ...postprocessing,
-    });
+  const {
+    data: { confusionMatrix: data, normalized } = {
+      confusionMatrix: [],
+      normalized: true,
```

Comment:
Interesting... We might want to better handle the loading and error states of this component in the future, but for now I think we should at least use our best guess as the default value: the query parameter.
```suggestion
      normalized: confusionMatrix.normalized,
```

Reply:
Yes, it sets the default value of `normalized` if the API response did not came back yet.

Reply:
Done in e819d2ccc535a4828cce530e4bab399a9724df11